README.md

```diff
@@ -1,19 +1,25 @@
 # triton-service-groups
 
+Triton Service Groups provide a simple way to scale instances to meet capacity needs and maintain performance at the lowest possible cost. Manage applications using the TSG API or with Terraform to scale VMs and containers on Triton. 
```

Comment:
This still mentions containers, are we certain folks will know the difference between infrastructure containers and Elastic Docker containers (which TSG does not support)?

Reply:
Yeah, we can't even talk about our Elastic Docker Host and bare metal Docker containers squarely.

Let's just say "add and remove instances" (not "scale" because that can include increasing their size, as well as number).

Reply:
> Triton Service Groups (TSGs) provide a simple way to manage any number of instances running the same image with the same configuration. Simply define the instance template, then create the service group and set the number of instances that are needed. As needs change, you can easily scale the number of instances.

Reply:
Updated